release_notes/RELEASE.txt

```diff
@@ -277,6 +277,19 @@ Support for new platforms and compilers
 
 Bugs fixed since HDF 4.2.15
 ===========================
+    - Correct theCMake generated pkg-config file
+
+      The pkg-config file generated by CMake had the order and placement of the
+      libraries wrong. Also added support for debug library names.
+
+      Changed the order of Libs.private libraries so that dependencies come after
+      dependents. Did not move the compression libraries into Requires.private
+      because there was not a way to determine if the compression libraries had
+      supported pkconfig files. Still recommend that the CMake config file method
+      be used for building projects with CMake.
+
+      (ADB - 2023/02/16 GH-1546,GH-2259)
```

Comment:
Were these GH numbers copied from HDF5?

Reply:
Yes I just merged the note and forgot to cut the hdf5 nums